Assets/GothicVR/Scripts/Manager/Settings/GameSettings.cs

```diff
@@ -1,14 +1,17 @@
-﻿namespace GVR.Manager.Settings
+﻿using System.Collections.Generic;
+
+namespace GVR.Manager.Settings
 {
     [System.Serializable]
     public class GameSettings
     {
         public string GothicIPath;
         public string GothicILanguage;
         public string LogLevel;
-        
+
         public string GothicMenuFontPath;
         public string GothicSubtitleFontPath;
 
+        public Dictionary<string, Dictionary<string, string>> GothicINISettings = new();
```

Comment:
Could you also load GothicGameIni settings too. I would love to use them to handle the other settings in the near future as well.
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/0efba5dd-6105-4c5f-a905-20bb632e9bb9)

Reply:
We can load GothicGame.ini, but i'm not sure that GOG version has this ini file as it seems to be a mod ini file from steam

Reply:
Oh ok. I created a ticket to check on this.
https://github.com/GothicVRProject/GothicVR/issues/298